content/content-strategy/identify-business-needs/who-to-talk/index.yml

```diff
@@ -0,0 +1,19 @@
+pagetitle: Who to talk to
+description: You’ll need to look at documents, run workshops and speak to different business areas.
+weight: 10
```

Comment:
Check the weight property. Is the same as multiple other subtopics in this module and will cause display issues.